bin/utils/java-utils.sh

```diff
@@ -63,7 +63,7 @@ validate_java_home() {
   fi
 
   # As we know the java -version command works then strip out the line we need
-  java_version_output=$(${JAVA_HOME}/bin/java -version 2>&1 | grep ^"java version")
+  java_version_output=$(${JAVA_HOME}/bin/java -version 2>&1 | grep ^"openjdk version")
```

Comment:
This will NOT work on z/os, even if it does work on linux.
What you really need is to test conditional to the OS that is being used, or try both if one fails